src/App.js

```diff
@@ -1,17 +1,16 @@
 import { Route, Routes } from 'react-router-dom';
-import Books from './components/Books';
 import Categories from './components/Categories';
 import Nav from './components/Navbar';
+import AddBookForm from './components/form';
 
 function App() {
   return (
     <>
       <Nav />
       <Routes>
-        <Route exact path="/" element={<Books />} />
+        <Route path="/form" element={<AddBookForm />} />
```

Comment:
- [ ] **[OPTIONAL]** I think you should maintain the initial root path for books you had. `/form` seems a bit off.

Reply:
thanks for the feedback @ [julie-ify](https://github.com/julie-ify), ill make the optional changes